.eslintrc.js

```diff
@@ -40,4 +45,16 @@ module.exports = {
       },
     ],
   },
+
+  overrides: [
+    {
+      files: [
+        '**/__tests__/*.{j,t}s?(x)',
```

Comment:
I added jest to the package, but didn't finally write any tests.

Reply:
Do you have any concrete proposals, regarding testing?

Is Jest really the best way to go about it? To be honest, I would personally suggest to look into something allowing us to write tests from a user's perspective, e.g. E2E or integration testing (like Selenium) in a browser, over anything like DOM-based testing. Also to take care of stuff like CSS hiding things based on viewport sizes etc.

Some resources:
* https://aswinkumar4018.medium.com/which-e2e-testing-framework-to-use-for-js-based-client-applications-fbcac9aab680
* https://www.netguru.com/blog/which-javascript-ui-testing-framework-to-use-in-2020

It seems that Good Old Selenium or Cypress are viable options.

It seems to me it would be wise contemplating this a little more before making a decision and, hence, leaving unused Jest-stuff out of this PR.

Reply:
No, I was just happy to have jest installed so I can make unit tests. I did not want to remove it again so next for the next occasion I have to install it again. If you prefer, I will write a unit test to fix the bug you spotted so that it has more merit to stay? :D 

For e2e it seems puppetteer is good. 

https://blog.logrocket.com/react-end-to-end-testing-jest-puppeteer/

Reply:
Now there is a test, so we stick with jest 👯‍♀️ Great if anyone cares to implement e2e testing